include/deal.II/lac/affine_constraints.h

```diff
@@ -1990,13 +1990,37 @@ class AffineConstraints : public Subscriptor
                             const bool                   verbose = false) const;
 
   /**
-   * Make the current object consistent on all processors
-   * in a distributed computation. One should call this function before
-   * calling close().
+   * Make the constraints that are locally stored on the current object
+   * consistent on all MPI processes in a distributed computation.
+   *
+   * The IndexSet @p locally_owned_dofs conforms to all DoFs that are locally
+   * owned on the current process, i.e., DoFHandler::locally_owned_dofs().
+   *
+   * The IndexSet @p constraints_to_make_consistent shall contain all DoF
+   * indices on which we want to store the complete constraints for. Which DoFs
```

Comment:
I am not sure in terms of the preposition, but I think it sounds better with
```suggestion
   * indices for which we want to store the complete constraints for. Which DoFs
```